java/vector/src/main/java/org/apache/arrow/vector/complex/impl/ComplexWriterImpl.java

```diff
@@ -159,6 +164,11 @@ public MapWriter rootAsMap() {
     return mapRoot;
   }
 
+  @Override
+  public MapWriter rootAsMap() {
+    return rootAsMap(null);
```

Comment:
nvm, i guess it doesn't matter if case sensitivity is configured when ComplexWriter is constructed.